Source/WebCore/page/ElementTargeting.cpp

```diff
@@ -332,7 +351,48 @@ Vector<TargetedElementInfo> findTargetedElements(Page& page, TargetedElementRequ
     Vector<TargetedElementInfo> results;
     results.reserveInitialCapacity(targets.size());
     for (auto iterator = targets.rbegin(); iterator != targets.rend(); ++iterator)
-        results.append(targetedElementInfo(*iterator));
+        results.append(targetedElementInfo(*iterator, true));
```

Comment:
Optional since this is a single boolean argument, but consider `enum class IsUnderPoint`

Reply:
Yeah, I agree the named enum makes it a bit clearer — will change to use `enum class IsUnderPoint `.

Thanks for the review!